base_layer/wallet/src/transaction_service/storage/memory_db.rs

```diff
@@ -250,6 +250,22 @@ impl TransactionBackend for TransactionMemoryDatabase {
             db.completed_transactions.contains_key(&tx_id))
     }
 
+    fn insert_completed_transaction(
```

Comment:
Don't need to add an explicit function to do this. Use the Write operation with the `DbKeyValuePair::CompletedTransaction(...,...)` It is already part of the unit tests etc.

Reply:
See previous comment.